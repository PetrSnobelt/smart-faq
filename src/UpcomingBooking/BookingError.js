// @flow

import * as React from 'react';
import css from 'styled-jsx/css';
import { Typography } from '@kiwicom/orbit-components';

const style = css`
  .UpcomingBooking {
    width: 480px;
    padding: 40px;
    background-color: #f5f7f9;
    height: 100%;
  }
  div.feedback {
    margin-top: 75%;
  }
  h1.title {
    font-family: Roboto;
    font-size: 28px;
    font-weight: bold;
    line-height: 100%;
    color: #171b1e;
    margin-bottom: 8px;
  }
`;

const BookingNotFound = () => {
  return (
    <div className="UpcomingBooking">
      <div className="feedback">
        <h1 className="title">Something&apos;s not right</h1>
        <Typography type="secondary" size="large">
          We weren&apos;t able to find your booking. Please, check your
          credentials and try again.
        </Typography>
      </div>
      <style jsx>{style}</style>
    </div>
  );
};

export default BookingNotFound;
