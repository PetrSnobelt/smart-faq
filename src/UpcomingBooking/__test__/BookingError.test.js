// @flow

import * as React from 'react';
import { shallow } from 'enzyme';

import BookingError from '../BookingError';

describe('BookingError', () => {
  const component = shallow(<BookingError />);
  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
