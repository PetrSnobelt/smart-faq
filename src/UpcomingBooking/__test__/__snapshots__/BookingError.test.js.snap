// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookingError should match snapshot 1`] = `
<div
  className="jsx-1862283839 BookingError"
>
  <h1
    className="jsx-1862283839 title"
  >
    Something's not right
  </h1>
  <Typography
    size="large"
    type="secondary"
  >
    We weren't able to find your booking. Please, check your credentials and try again.
  </Typography>
  <JSXStyle
    css=".BookingError.jsx-1862283839{width:480px;padding:40px;background-color:#f5f7f9;height:100%;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;}h1.title.jsx-1862283839{font-family:Roboto;font-size:28px;font-weight:bold;line-height:100%;color:#171b1e;margin-bottom:8px;}"
    styleId="1862283839"
  />
</div>
`;
